ecocodecsharp.properties

```diff
@@ -12,3 +12,9 @@ defaults.mavenArtifactId=ecocode-csharp-plugin
 1.1.0.date=2024-05-26
 1.1.0.downloadUrl=https://github.com/green-code-initiative/ecoCode-csharp-sonarqube/releases/download/1.1.0/ecocode-csharp-plugin-1.1.0.jar
 1.1.0.changelogUrl=https://github.com/green-code-initiative/ecoCode-csharp-sonarqube/releases/tag/1.1.0
+
+1.5.1.description=Add rules EC91, EC92, EC93; support of SonarQube 10.7; upgrade internal library versions
+1.5.1.sqVersions=[9.9.5,LATEST]
```

Comment:
Ideally this range will start with `9.9`. As is, you've locked people on 9.9(.0), 9.9.1 &etc, out of installing via the Marketplace. But it's up to you. (And yes, I should have noticed & said something when you published 1.1.0.)

Reply:
Thx for this review, I have fixed the range on v1.5.1 and v1.1.0 too, even if archived.
Please let me know if it'ok to modify a previous version in a properties file.